CHANGELOG.adoc

```diff
@@ -38,7 +38,8 @@ Rename the class `SemanticCheckerFactory` to `SemanticRunnableFactory` to reflec
 - [releng] Switch to `commons-io 2.17` 
 
 === Bug fixes
-- https://github.com/eclipse-syson/syson/issues/744[#744] [import] Fix containment of imported Allocation and AllocationDefinition
+- https://github.com/eclipse-syson/syson/issues/744[#744] [import] Fix containment of imported Allocation and AllocationDefinition.
+- https://github.com/eclipse-syson/syson/issues/777[#777] [import] Fix import of some boolean properties of elements.
```

Comment:
Could you please list the attributes you're talking about and on which Element they are associated? Type, Feature, Definition, something else?
Could you also change the release notes as well?

Reply:
done

Reply:
Sorry but your didn't take into account my remark for the types. For example, `isAbstract` is applied on `Type`. So you could write `Type#isAbstract` or something like that instead of just `isAbstract`.
Could you add them please?